Assets/AirshipPackages/@Easy/Core/Shared/Camera/CameraSensitivity.ts

```diff
@@ -0,0 +1,15 @@
+import { Airship } from "../Airship";
+
+const BASE_DPI = 96;
+
+/** Get mouse sensitivity, corrected for the user's screen DPI. */
+export function getDpiAdjustedMouseSensitivity(): number {
+	const dpi = Screen.dpi;
```

Comment:
This value won't change, assuming the current screen doesn't change. So this could probably be cached. But then again, that wouldn't work correctly if the user changes their screen. E.g. if they have two monitors with different DPIs, and then move Airship from window to another, then a cached DPI value here would get messed up.

Since we don't have screen switch detection, I vote to just leave this as-is for now.